src/components/Strategies/StrategyList.tsx

```diff
@@ -1,59 +1,39 @@
 import { Card, Row, Col, Typography } from 'antd';
 import styled from 'styled-components';
 
+import { IStrategies } from './../../types';
+
```

Comment:
Remove this empty line. Let's adhere to the convention of 'external imports; empty line; internal imports'